#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_asn2wrs_ply.dpatch by Jakub Wilk <ubanus@users.sf.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: use lex/yacc from ply

@DPATCH@
diff --git a/tools/asn2wrs.py b/tools/asn2wrs.py
--- a/tools/asn2wrs.py
+++ b/tools/asn2wrs.py
@@ -47,8 +47,8 @@
 import getopt
 import traceback
 
-import lex
-import yacc
+from ply import lex
+from ply import yacc
 
 # OID name -> number conversion table
 oid_names = {
